tests/helpers/adminMenu.js

```diff
@@ -0,0 +1,12 @@
+module.exports.open = function (client) {
+  const menuBtnXPathSelector = '//div[@data-apos-actionable="data-apos-admin-bar"]';
+  const tagXPathSelector = '(//*[contains(@class, "apos-admin-bar-item-inner")])[8]';
```

Comment:
It looks like "workflow" menu item but not like whole admin bar.

Reply:
+